common/validators.js

```diff
@@ -31,6 +31,38 @@
         };
     })
 
+    .directive('chaiseMin', function() {
+        return {
+            require: 'ngModel',
+            link: function(scope, elm, attrs, ctrl) {
+                ctrl.$validators.chaiseMin = function(modelValue, viewValue) {
+                    if (ctrl.$isEmpty(modelValue)) {
+                        // consider empty models to be valid
+                        // use the `required` attribute in the HTML
+                        return true;
+                    }
+                    return parseInt(viewValue, 10) >= parseInt(attrs.chaiseMin, 10);
+                };
+            }
+        }
+    })
+
+    .directive('chaiseMax', function() {
+        return {
+            require: 'ngModel',
+            link: function(scope, elm, attrs, ctrl) {
+                ctrl.$validators.chaiseMax = function(modelValue, viewValue) {
+                    if (ctrl.$isEmpty(modelValue)) {
+                        // consider empty models to be valid
+                        // use the `required` attribute in the HTML
+                        return true;
+                    }
+                    return parseInt(viewValue, 10) <= parseInt(attrs.chaiseMax, 10);
+                };
+            }
+        }
+    })
+
     // TODO: this is not working as expected, 1.1.1 is "valid"
```

Comment:
Does this bug still exist with the changes in this PR? If not, you should update this comment. 

You might want to update the last line that explains how to use it. Although technically that's still correct.

Reply:
Right, I will remove this comment and update the other one.